components/my-account/User/UserForm/UserForm.tsx

```diff
@@ -0,0 +1,368 @@
+          <Grid
+            item
+            xs={12}
+            md={3.5}
+            sx={{ paddingTop: { xs: '10px !important', md: '64px' } }}
+            style={{ display: 'flex', alignItems: 'center' }}
+          >
+            <Controller
+              name="emailAddress"
+              control={control}
+              rules={{
+                pattern: {
+                  value: /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i,
+                  message: t('invalid-email-error'),
+                },
+              }}
+              render={({ field }) => (
+                <KiboTextBox
+                  fullWidth
+                  error={!!errors?.emailAddress}
+                  value={field.value || ''}
+                  helperText={errors?.emailAddress?.message}
+                  label={t('email-address')}
+                  onChange={(_name, value) => field.onChange(value)}
+                />
+              )}
+            />
+          </Grid>
+          <Grid
+            item
+            xs={12}
+            md={1.5}
+            sx={{ paddingTop: { xs: '10px !important', md: '64px' } }}
+            style={{ display: 'flex', alignItems: 'center' }}
+          >
+            <Controller
+              name="firstName"
+              control={control}
+              render={({ field }) => (
+                <KiboTextBox
+                  fullWidth
+                  error={!!errors?.firstName}
+                  value={field.value || ''}
+                  helperText={errors?.firstName?.message}
+                  label={t('first-name')}
+                  onChange={(_name, value) => field.onChange(value)}
+                />
+              )}
+            />
+          </Grid>
+          <Grid
+            item
+            xs={12}
+            md={1.5}
+            sx={{ paddingTop: { xs: '10px !important', md: '64px' } }}
+            style={{ display: 'flex', alignItems: 'center' }}
+          >
+            <Controller
+              name="lastName"
+              control={control}
+              render={({ field }) => (
+                <KiboTextBox
+                  fullWidth
+                  error={!!errors?.lastName}
+                  value={field.value || ''}
+                  helperText={errors?.lastName?.message}
+                  label={t('last-name-or-sur-name')}
+                  onChange={(_name, value) => field.onChange(value)}
+                />
+              )}
+            />
+          </Grid>
+          <Grid item xs={12} md={2} sx={{ paddingTop: { xs: '10px !important', md: '64px' } }}>
+            <Controller
+              name="role"
+              control={control}
+              defaultValue={getValues('role')}
+              render={({ field }) => (
+                <KiboRadio
+                  {...field}
+                  align="center"
+                  onChange={(value) => field.onChange(value)}
+                  title={mdScreen ? '' : 'Role'}
```

Comment:
Let's add the roles title on Desktop as well for consistency

Reply:
Done